docs/site/Decorators.md

```diff
@@ -421,62 +426,31 @@ The format of property definitions can be found in
 
 For usage examples, see [Define Models](Repositories.md#define-models)
 
-### Relation Decorators
-
-The relation decorator defines the nature of a relationship between two models.
-
-_This feature has not yet been released in alpha form. Documentation will be_
-_added here as this feature progresses._
-
-#### Relation Decorator
-
-Syntax: `@relation`
-
-Register a general relation.
-
-#### Specfic Relation Decorator
-
-Syntax:
-
-- `@belongsTo`
-- `@hasOne`
-- `@hasMany`
-- `@embedsOne`
-- `@embedsMany`
-- `@referencesOne`
-- `@referencesMany`
-
-Register a specific relation
-
 ### Repository Decorator
 
 Syntax:
-
-```ts
-@repository(model: string | typeof Entity, dataSource?: string | juggler.DataSource)
-```
+`@repository(model: string | typeof Entity, dataSource?: string | juggler.DataSource)`
```

Comment:
should we link to source code/apidocs here as well?

Reply:
done